frontend/src/app/containers/ExperimentPage/index.js

```diff
@@ -878,46 +437,33 @@ export class ExperimentDetail extends React.Component {
     });
 
     disableExperiment(experiment);
-  }
+  };
 
-  renderUninstallSurvey(evt) {
+  doShowTourDialog = (evt: MouseEvent) => {
     evt.preventDefault();
+    this.setState({ showTourDialog: true });
+  };
 
-    this.uninstallExperiment(evt);
+  doShowEolDialog = (evt: MouseEvent) => {
```

Comment:
This is new stuff: read [a Medium post about binding functions for React](https://medium.freecodecamp.org/react-binding-patterns-5-approaches-for-handling-this-92c651b5af56) and thought this class arrow function syntax would be cleaner for passing these utilities down into sub-components.